front/src/applications/stdcm/components/StdcmForm/StdcmConsist.tsx

```diff
@@ -65,6 +66,7 @@ const StdcmConsist = ({
   disabled = false,
   consistErrors,
   setConsistErrors,
+  displayLoadingGaugeSelect = true,
```

Comment:
nit : We try not to have props defaulted to `true`.


```suggestion
  hideLoadingGaugeSelector = false,
```

Reply:
I reworked this part a bit so this field does not exist anymore as a paramenter. It is defined as a variable in the component now